narayana/stm/src/main/java/org/jboss/narayana/stm/BaselineBenchmark.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2014, Oracle America, Inc.
```

Comment:
Copyright change

Reply:
Sorted. It comes from the command:
```
mvn archetype:generate -DarchetypeGroupId=org.openjdk.jmh -DarchetypeArtifactId=jmh-java-benchmark-archetype
```
which is a bit annoying, it will be generated by including -DarchetypeArtifactId=jmh-java-benchmark-archetype